### PR TITLE
Patch 4

### PR DESCRIPTION
**Pull request by Ioxa53**

### CODE REVIEW

File: Shaders/HighPassSharpen.fx [hunk 1]
```diff
@@ -14,163 +13,112 @@ uniform float HighPassSharpOffset <
 	ui_type = "drag";
 	ui_min = 0.00; ui_max = 1.00;
 	ui_tooltip = "Additional adjustment for the blur radius. Values less than 1.00 will reduce the radius limiting the sharpening to finer details.";
-	ui_step = 0.20;
 > = 1.00;
 
 uniform int HighPassBlendMode <
```
Comment (crosire) [inline Shaders/HighPassSharpen.fx]: I think a combo box is better suited here. So `ui_type = "combo"` and `ui_items = "Soft Light\0Overlay\0Mulitply\0Hard Light\0Vivid Light\0Screen\0Linear Light"`. Only that the value would go from 0 to 6 then and not 1 to 7.

Comment (Ioxa53) [inline Shaders/HighPassSharpen.fx]: I agree, that is much better to work with. I made use of the combo box in the Clarity shader as well.

**Comment by crosire** [comment]
@Ioxa53 I just realized that the "bilateral" on the shadertoy filter actually refers to that you can move the active zone around and has nothing to do with the blurring itself. So the name "BilateralBlur" is really misleading, because that's not what it does.

**Comment by Ioxa53** [comment]
Would "SelectiveBlur" or "SurfaceBlur" be better?

**Comment by crosire** [comment]
I think so, yes. SurfaceBlur describes it nicely.

**Pull request merged**